core/src/main/resources/reference.conf

```diff
@@ -167,6 +167,11 @@ system {
 
   # Rate at which Cromwell updates it's instrumentation gauge metrics (e.g: Number of workflows running, queued, etc..)
   instrumentation-rate = 5 seconds
+  
+  job-rate-control {
```

Comment:
ToL: I dislike the generic namespace we use `system.*`. Someday we'll refactor them to something like `cromwell.system.*` to avoid conflicts.